device_calendar/example_app/lib/presentation/pages/calendar_events.dart

```diff
@@ -81,7 +81,7 @@ class _CalendarEventsPageState extends State<CalendarEventsPage> {
               new MaterialPageRoute(builder: (BuildContext context) {
             return new CalendarEventPage(_calendar);
           }));
-          if (refreshEvents) {
+          if (refreshEvents == true) {
```

Comment:
Is an explicit comparison required in here?

Reply:
yes Dart types are nullable

Reply:
👍